src/win/tty.c

```diff
@@ -2168,3 +2191,24 @@ int uv_tty_reset_mode(void) {
   /* Not necessary to do anything. */
   return 0;
 }
+
+/* Determine whether or not this version of windows supports
+ * proper ANSI color codes. Should be supported as of windows
+ * 10 version 1511, build number 10.0.10586.
+ */
+void uv__determine_vterm_state(HANDLE handle) {
+  DWORD dwMode = 0;
+
+  if (!GetConsoleMode(handle, &dwMode)) {
+      vterm = UNSUPPORTED;
```

Comment:
As far as I can tell we use 2 space indents everywhere in this file as well as others eg. src/unix/tty.c
Should I be using tabs instead?